components/event_dispatcher/introduction.rst

```diff
@@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
 ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
 e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
 avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
-il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
+il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
 nell'oggetto distributore. Ecco degli esempi:
```

Comment:
Quanti di questi riportano la dicitura 'caricamento pigro' esclusivamente
in italiano senza riportare l'equivalente inglese?
Il 22/mar/2015 22:10 "Massimiliano Arione" <notifications@github.com> ha
scritto:
>
> In components/event_dispatcher/introduction.rst:
>
> > @@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
> >  ``EventDispatcher`` inietta sempre l'evento distribuito, il nome
dell'evento
> >  e un riferimento a sé stesso agli ascoltatori. Questo può portare ad
applicazioni
> >  avanzate per ``EventDispatcher``, incluse la possibilità per gli
ascoltatori di distribuire altri eventi,
> > -il concatenamento degli eventi o anche il caricamento pigro di più
ascoltatori
> > +il concatenamento degli eventi o anche il caricamento pigro ("lazy")
di più ascoltatori
> >  nell'oggetto distributore. Ecco degli esempi:
>
> La SERP (rigorosamente su scheda anonima) di Google per "caricamento
pigro" restituisce pagine sul lazy loading in 8 dei primi 10 risultati.
>
> —
> Reply to this email directly or view it on GitHub.